### PR TITLE
121630-update-app-insights

### PR DESCRIPTION
**Pull request by simonnineteen90**
https://dev.azure.com/defragovuk/DEFRA-FFC/_sprints/taskboard/DEFRA-FFC%20Payments%20Team/DEFRA-FFC/Payment%20Service/Pay%20Sprint%2048?workitem=121630

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,7 +1,7 @@
 {
   "name": "ffc-demo-payment-web",
   "description": "Digital service mock to view scheduled payments for financial aid in the event a property subsides into mine shaft.",
-  "version": "1.19.4",
+  "version": "1.19.5",
   "homepage": "https://github.com/DEFRA/ffc-demo-payment-web",
   "repository": {
     "type": "git",
```
Comment (marctemp) [inline package.json:7]: AppInsights here is version `1.8.2` we're using `2.3.3` in other repos -- let's update

Comment (marctemp) [reply package.json:7]: Be worth checking the other dependencies against our de-facto versions in other repos

Comment (simonnineteen90) [reply package.json:7]: compared vs payment service repo and updated

**Comment by marctemp** [comment]
Can we add a unit test file around the appInsights setup?

Have done likewise in `ffc-pay-web`: https://github.com/DEFRA/ffc-pay-web/blob/121727/switch-app-insights-conn-method/test/unit/insights.test.js

**Review by marctemp — APPROVED** [review]
Looks delicious

**Pull request merged**